src/app/_components/Slider.js

```diff
@@ -0,0 +1,21 @@
+'use client';
+
+import PropTypes from 'prop-types';
+import { Swiper } from 'swiper/react';
+
+import 'swiper/css';
+
+export function Slider({ slidesPerView = 'auto', spaceBetween = 15, children, className }) {
```

Comment:
You can keep slidePerView and spaceBetween here if you want to give them default values, but I would also add all others swiper attributes using spread operator, e.g children, className, ...swiperProps }) and later in code <Swiper ...currentAttributes, {...swiperProps} />

Reply:
Updated Slider component as per your comments